jc/parsers/amixer.py

```diff
@@ -0,0 +1,277 @@
+    $ amixer sget Capture | jc --amixer
+    $ amixer sget Speakers | jc --amixer
+
+Usage (module):
+
+    import jc
+    result = jc.parse('amixer', <amixer sget command output>)
+
+Schema:
+
+    {
+        "control_name":                     string,
+        "capabilities": [
+                                            string
+        ],
+        "playback_channels": [
+            string
+        ],
+        "limits": {
+            "playback_min":                 integer,
+            "playback_max":                 integer
+        },
+        "mono": {
+            "playback_value":               integer,
+            "percentage":                   integer,
+            "dB":                           float,
+            "status":                       boolean
+        }
+    }
+
+Examples:
+
+    $ amixer sget Master | jc --amixer -p
+    {
+      "control_name": "Capture",
+      "capabilities": [
+        "cvolume",
+        "cswitch"
+      ],
+      "playback_channels": [],
+      "limits": {
+        "playback_min": 0,
+        "playback_max": 63
+      },
+      "front_left": {
+        "playback_value": 63,
+        "percentage": 100,
+        "dB": 30.0,
+        "status": true
+      },
+      "front_right": {
+        "playback_value": 63,
+        "percentage": 100,
+        "dB": 30.0,
+        "status": true
+      }
+    }
+
+    $ amixer sget Master | jc --amixer -p -r
+    {
+        "control_name": "Master",
+        "capabilities": [
+            "pvolume",
+            "pvolume-joined",
+            "pswitch",
+            "pswitch-joined"
+        ],
+        "playback_channels": [
+            "Mono"
+        ],
+        "limits": {
+            "playback_min": "0",
+            "playback_max": "87"
+        },
+        "mono": {
+            "playback_value": "87",
+            "percentage": "100%",
+            "dB": "0.00dB",
+            "status": "on"
+        }
+    }
+
+
+"""
+from typing import List, Dict
+
+import jc.utils
+
+
+class info():
+    """Provides parser metadata (version, author, etc.)"""
+    version = '1.0'
+    description = '`amixer` command parser'
+    author = 'Eden Refael'
+    author_email = 'edenraf@hotmail.com'
+    compatible = ['linux']
+    magic_commands = ['amixer']
+    tags = ['command']
+
+
+__version__ = info.version
+
+
+def _process(proc_data: dict) -> dict:
+    """
+    Processes raw structured data to match the schema requirements.
+
+    Parameters:
+        proc_data: (dict) raw structured data from the parser
+
+    Returns:
+        (dict) processed structured data adhering to the schema
+    """
+    # Initialize the processed dictionary
+    processed = {
+        "control_name": proc_data.get("control_name", ""),
+        "capabilities": proc_data.get("capabilities", []),
+        "playback_channels": proc_data.get("playback_channels", []),
+        "limits": {
+            "playback_min": int(proc_data.get("limits", {}).get("playback_min", 0)),
```

Comment:
Just an FYI you could also use the `jc.utils.convert_to_int`, etc. which should do a lot of the text removal for you.